prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -123,10 +123,13 @@ size_t rxr_pkt_req_data_size(struct rxr_pkt_entry *pkt_entry)
 	return pkt_entry->pkt_size - rxr_pkt_req_data_offset(pkt_entry);
 }
 
-int rxr_pkt_type_readbase_rtm(int op, uint64_t fi_flags)
+int rxr_pkt_type_readbase_rtm(struct rdm_peer *peer, int op, uint64_t fi_flags)
 {
 	assert(op == ofi_op_tagged || op == ofi_op_msg);
-	if (rxr_env.efa_runt_size > 0 && !(fi_flags & FI_DELIVERY_COMPLETE)) {
+	if (cuda_is_gdrcopy_enabled() &&
```

Comment:
nit: this `cuda_is_gdrcopy_enabled` check may should be inside the commit that do runt for cuda only?

Reply:
Yes. I think so.